tox.ini

```diff
@@ -1,10 +1,12 @@
 [tox]
-envlist = py27, pypy
+envlist = py27, pypy, py34
 
 [testenv]
 deps = 
     pytest
     pytest-capturelog
+    tzlocal
+    configobj
```

Comment:
I'm not sure what you mean -- tox installs the current pkg into the created virtualenv by default, AFAIK